tests/php/ORM/DataQueryTest.php

```diff
@@ -879,4 +879,38 @@ public function testWithUsingDataQueryAppliesRelations()
         // This will throw an exception if it fails - it passes if there's no exception.
         $dataQuery->execute();
     }
+
+    public function provideGetLimit(): array
```

Comment:
Ideally we'd get rid of this test - see my comment on DataQuery

Reply:
Done